### PR TITLE
Updates to Privacy Sandbox article

### PR DESCRIPTION
**Pull request by samdutton**

### CODE REVIEW

**Comment by samdutton** [comment]
I think this should have a once-over from a tech writer, and a quick check from others as per the email thread.

Once that's done, we can remove the DO NOT MERGE label.

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md [hunk 1]
```diff
@@ -2,195 +2,197 @@
...
 description: The Privacy Sandbox is a series of proposals to satisfy third-party use cases without third-party cookies or other tracking mechanisms. The proposals need your feedback.
 tags:
-  - blog
-  - privacy
-  - security
-  - trust and safety
+- post
+- privacy
+- security
+- trust and safety
```
[the remark this answers is not in the document]
Comment (agektmr) [reply src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md:16]: Is there a list of available / recommended tags for web.dev?

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md [hunk 2]
```diff
@@ -2,195 +2,197 @@
...
 
-Browsers have taken steps to [deprecate third-party cookies](https://blog.chromium.org/2020/01/building-more-private-web-path-towards.html?mod=article_inline), but techniques to identify and track the behaviour individual users, known as fingerprinting, have continued to evolve. Fingerprinting uses mechanisms that users aren't aware of and can't control.
+**Goal:** Reduce the amount of potentially identifiable data revealed by APIs and make access to potentially identifiable data controllable by users, and measurable.
 
-{% Aside %}
-Sites such as [Panopticlick](https://panopticlick.eff.org) and [amiunique.org](https://amiunique.org/) show how fingerprint data can be combined to identify you as an individual.
-{% endAside %}
+Browsers have taken steps to [deprecate third-party cookies](https://blog.chromium.org/2020/01/building-more-private-web-path-towards.html?mod=article_inline), but techniques to identify and track the behaviour of individual users, known as fingerprinting, have continued to evolve. Fingerprinting uses mechanisms that users aren't aware of and can't control.
```
[the remark this answers is not in the document]
Comment (samdutton) [reply src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md:127]: No! Good catch. Removed in another commit.

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md [hunk 3]
```diff
@@ -2,195 +2,197 @@
...
 
-Ads play a major role in making the internet an easily accessible, open place, but advertising has to work for everyone. When it does, it means users can get access to information and great content, creators and publishers can thrive by getting fair compensation for their work, and marketers can connect with people interested in what they have to offer. Which brings us to our goal: to evolve the web to better respect user privacy and support a vibrant, thriving ecosystem.
+In particular, how can websites fund content by enabling third parties to show ads and measure ad performance—but not allow individual users to be profiled? How can advertisers and site owners evaluate a user's authenticity without resorting to dark patterns such as device fingerprinting?
 
-## The current state of privacy on the web
+The way things work at the moment can be problematic for the entire web ecosystem, not just users. For publishers and advertisers, tracking identity and using a variety of non-standard third-party solutions can add to technical debt, code complexity and data risk. Users, developers, publishers, and advertisers should be confident that the web is protecting user privacy choices.
```
[the remark this answers is not in the document]
Comment (samdutton) [reply src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md:41]: 👍

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (samdutton) [reply src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md]: BTW @kaycebasques: only picked up on **Add suggestion to batch** after committing a few. This is much better (and with Tom's suggestion).

**Comment by kaycebasques** [comment]
@samdutton you can merge this yourself if it's ready

**Pull request merged**